webapp-admin/public/index.html

```diff
@@ -7,15 +7,11 @@
 	<title>{{.Title}}</title>
 
 	<link rel="stylesheet" href="/static/font-awesome/css/font-awesome.min.css" />
-	<link rel="stylesheet" href="/static/css/application.css" />
```

Comment:
the new built js/css files look like `main.652bcdd4.chunk.js` and are automatically added to this html file. So we don't need these hardcoded static names here.